ANS_JS_template.tex

```diff
@@ -208,88 +220,35 @@ \section{Nuclear Fuel Cycle Simulators}
+
+The 2016 Electric Power Research Institute report "Program on Technology Innovation: Assessment of Nuclear Fuel Cycle Simulation Tools" takes a commercial software architecture approach to analyze sixteen NFCSs.  The NFCSs are evaluated based on functionality, usability, reliability, performance, and supportability. Of the sixteen NFCSs included, only six appeared to be currently in development: CAFCA, COSI6, CYCLUS, DANESS, ORION, and VISION. As is true with other NFCS Benchmarking papers, the EPRI report used feedback from users of the NFCS \cite{EPRI2016}.  The data in the report is anonymous, making it difficult to distinguish which of the NFCSs particularly stood out for each of the categories. The overall approach of analyzing the software based on commercial software architecture criteria, with many subcriteria, is an important assessment for the adoption of the tool by end users. Some of the criteria the end users involved in the survey viewed as being helpful included: parallel processing, open-source, the flexibility to model complex scenarios, uncertainty/sensitivity analysis, economic capabilities, visual tools, and ability to track many nuclides.
 
-Juchau (2010) reviews the nuclear fuel cycle simulator tools CAFCA, COSI, CEPMNFC, DANESS, NFCSim, VEGAS, VISION, NFCSS, and GENIUS 1. His conclusion is that many of these tools have overlapping functionalities, partially due to the proprietary nature of at least parts of each. One of the reasons why there are so many nuclear fuel cycle simulators is that many of the countries that currently have NPPs have designed their own simulators. Figure 3 depicts the non-US NFCSs along with some of their software dependencies. Most of the fuel cycle simulators are not open source or available for download; thus, each nation tends to have proprietary software. As discussed with Prof. Kathryn D. Huff, it is also often more efficient, both in terms of time spent as well as better integration, if the different functions are designed by the same team (\cite{redfoot_huff_2016}). Integrating components, such a burnup calculation, from one system into a new system can often be more challenging than building the function from scratch. While sharing code can be challenging amongst various program, some calculations or libraries are commonly used by NFCSs. For example, Figure 3 depicts EVOLCODE, FUTURE, and FAMILY21 as all relying on Origen.
+From the literature reviewed and figures 2a and 2b, nuclear fuel cycle simulators appear to have certain common characteristics. Generally, there is a means of tracking the nuclear fuel materials and the composition of those materials using an associated decay software. NFCSs should generally be flexible and able to include new components, able to incorporate other pieces of software, able to do an uncertainty analysis, and accessible to technical and non technical audiences alike.
```

Comment:
Again. They have the common characteristics.

Reply:
Do you think I should take this part out?